hphp/hack/src/stubs/state_loader_lwt.ml

```diff
@@ -6,20 +6,10 @@
  *
  *)
 
-let get_project_metadata
-    ~progress_callback:_
-    ~saved_state_type:_
-    ~repo:_
-    ~ignore_hh_version:_
-    ~opts:_ =
+let get_project_metadata ~repo:_ ~ignore_hh_version:_ ~opts:_ =
   failwith "Not implemented"
 
-let load
-    ~ssopt:_
-    ~progress_callback:_
-    ~watchman_opts:_
-    ~ignore_hh_version:_
-    ~saved_state_type:_ =
+let load ~ssopt:_ ~progress_callback:_ ~watchman_opts:_ ~ignore_hh_version:_ =
   failwith "Not implemented"
```

Comment:
This needs to return `Lwt.return (Error "Not implemented")` for the IDE integration to work